src/recommenders/bi.py

```diff
@@ -26,27 +26,30 @@ def _recommend(self):
             score = self._objective(config)
             self.scores.append(score)
 
-        for i in range(6):
+        for i in range(2):
```

Comment:
Incorrect. Fix.

Reply:
Actually: these should be constants defined in `BaseRecommender`. Why use magic numbers at all?